machine_learning_hep/analysis/analyzer_jet.py

```diff
@@ -332,18 +391,23 @@ def fit(self):
                 histomass_reb.Copy(histomass_reb_f)
                 fitter = AliHFInvMassFitter(histomass_reb_f, self.p_massmin[ipt], \
                     self.p_massmax[ipt], self.p_bkgfunc[ipt], self.p_sgnfunc[ipt])
-                fitter.SetInitialGaussianSigma(fittermc.GetSigma())
-                fitter.SetInitialGaussianMean(fittermc.GetMean())
-                if self.p_fix_sigma[ipt] is True:
-                    fitter.SetFixGaussianSigma(fittermc.GetSigma())
+                fitter.SetInitialGaussianSigma(fittermc.GetParameter(2))
+                fitter.SetInitialGaussianMean(fittermc.GetParameter(1))
+                if self.p_set_initial_sigma[ipt] is True:
```

Comment:
lines 394 & 396-402:

if self.p_set_initial_sigma[ipt] is True:
    initial_sigma = self.p_sigmaarray[ipt]
else :
   initial_sigma = fittermc.GetParameter(2)

if self.p_set_fix_sigma[ipt] is True:
   fitter.SetFixGaussianSigma(initial_sigma)

Reply:
this wasnt implemented?